public/scss/main.scss

```diff
@@ -0,0 +1,14 @@
+@import "./abstracts/variables.scss";
+@import "./abstracts/mixins.scss";
+@import "./base/index.scss";
```

Comment:
Rename all these partials. You should begin with an underscore. `main.scss` should be the only file without underscore.